docs/AutomationReference.md

```diff
@@ -280,12 +292,25 @@ These versions embed the symbols into the assemblies and support [SourceLink](ht
 
 #### Error handling
 
-`AxeWindowsAutomationException` is thrown for all unhandled errors in Axe.Windows.Automation. If an exception was thrown from code not owned by Axe.Windows.Automation, that exception will be wrapped in the `Exception.InnerException` property.
+`AxeWindowsAutomationException` is thrown for errors in `Axe.Windows.Automation`'s logic where meaningful error reporting can be generated. In some situations, the `Exception.InnerException` property may contain a corresponding system-level exception for errors encountered by Axe.Windows.
+
+#### Migrating from Axe.Windows 1.x
+Migration to Axe.Windows 2.0 from 1.x should require minimal code changes for most projects.
```

Comment:
nit: Since people will be migrating *from* 1.x *to* 2.0, should we describe it that way?
```suggestion
Migration from Axe.Windows 1x to Axe.Windows 2.0 should require minimal code changes for most projects:
```